### DISCUSSION TITLE
Welcome to passcore Discussions!

### DISCUSSION THREAD

**Discussion by geoperez** [unosquare/passcore#610] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sergiochoo** [comment]
Has anybody made "Passcore" work in Docker?
I am trying to do this for two months, but no luck.
It doesn't work in Docker, at least free version, I guess.

**Reply by geoperez** [reply]
Do you have a specific error?

**Comment by Bmalone100** [comment]
Hi, I'm a support engineer with amcsgroup, I deal with external customers who use Passcore to change their Azure account passwords.

They often enter their full email when resetting their password which triggers an error 'Account is disable' - however entering the username only i.e. 'brian.malone' succeeds. 

Is it possible to add validation in the case where a user relentlessly adds their email to the username field even when specifically advised to enter their username with no email extension.

Split at '@' and take only the string value before the '@' sign.

This will greatly assist my department in reducing tickets related to user passwords.

Entered 'brian.malone@amcs.local:
![image](https://user-images.githubusercontent.com/36305546/211586866-573e2212-210f-4dda-8504-2a9aa1fb5cc6.png)

Entered 'brian.malone':
![image](https://user-images.githubusercontent.com/36305546/211587036-de45a48b-1dbb-469f-9d08-f9a5e4cfa610.png)

**Reply by geoperez** [reply]
Hey @Bmalone100, this discussion board is for passcore open source. For commercial one, please reach lorenzo.infante@unosquare.com.

Thanks
